src/controller/java/CHIPDeviceController-JNI.cpp

```diff
@@ -423,14 +520,14 @@ void HandleSimpleOperationComplete(ChipDeviceController * deviceController, void
     jobject self   = (jobject) deviceController->AppState;
     CHIP_ERROR err = CHIP_NO_ERROR;
 
-    ChipLogProgress(Controller, "Received response to %s request", (const char *) appReqState);
+    ChipLogProgress(Controller, "Received response to %s request", operation);
```

Comment:
Shouldn't we make sure before using that "operation" is not null?

Reply:
The function is only called from this module, so I guess it's safe to assume that it's not null

Reply:
Ok, nevermind then.